### PR TITLE
Fix softmax identification in model_loss

### PR DESCRIPTION
**Pull request by carlini**
model_loss determines softmax by looking for the string 'softmax' in the inputs to the tensor. Inception (and others) uses 'pre_softmax' for the logits, breaking it. This is a specific workaround for those default models.

### CODE REVIEW

**Comment by carlini** [comment]
Actually, I decided this probably is a bad idea. Let's not do this. It's better to have people just use the model interface.
